tools/CouchDB_Import_RadiologicalReview.php

```diff
@@ -1,85 +1,134 @@
+            'Type'        => "enum('No','Yes')",
+        ],
+        'ExtraReview_Radiologist'        => [
             'Description' => 'Radiologist/Reviewer doing the extra review',
-            'Type' => 'varchar(255)'
-        ),
-        'ExtraReview_Done' => array(
+            'Type'        => 'varchar(255)',
+        ],
+        'ExtraReview_Done'               => [
             'Description' => 'Extra review done',
-            'Type' => "enum('No','Yes')"
-        ),
-        'ExtraReview_Results' => array(
+            'Type'        => "enum('No','Yes')",
+        ],
+        'ExtraReview_Results'            => [
             'Description' => 'Results of the extra radiology review',
-            'Type' => "enum('normal','abnormal','atypical','not_answered')"
-        ),
-        'ExtraReview_ExclusionaryStatus' => array(
+            'Type'        => "enum('normal','abnormal','atypical','not_answered')",
+        ],
+        'ExtraReview_ExclusionaryStatus' => [
             'Description' => 'Extra review exclusionary status',
-            'Type' => "enum('exclusionary','non_exclusionary','not_answered')"
-        ),
-        'ExtraReview_SAS' => array(
+            'Type'        => "enum(
+				'exclusionary',
+				'non_exclusionary',
+				'not_answered'
+			     )",
+        ],
+        'ExtraReview_SAS'                => [
             'Description' => 'Extra review subarachnoid space',
-            'Type' => "enum('None', 'Minimal', 'Mild', 'Moderate', 'Marker')"
-        ),
-        'ExtraReview_PVS' => array(
+            'Type'        => "enum(
+				'None', 
+				'Minimal', 
+				'Mild', 
+				'Moderate', 
+				'Marker'
+			     )",
+        ],
+        'ExtraReview_PVS'                => [
             'Description' => 'Extra review perivascular space',
-            'Type' => "enum('None', 'Minimal', 'Mild', 'Moderate', 'Marker')",
-        ),
-        'ExtraReview_Comment' => array(
+            'Type'        => "enum('None', 'Minimal', 'Mild', 'Moderate', 'Marker')",
+        ],
+        'ExtraReview_Comment'            => [
             'Description' => 'Current stage of visit',
-            'Type' => "enum('Not Started','Screening','Visit','Approval','Subject','Recycling Bin')"
-        ),
-        'Conflict_Final_Extra' => array(
+            'Type'        => "enum(
+				'Not Started',
+				'Screening',
+				'Visit',
+				'Approval',
+				'Subject',
+				'Recycling Bin'
+			     )",
+        ],
+        'Conflict_Final_Extra'           => [
             'Description' => 'Conflict between final and extra reviews',
-            'Type' => "enum('No','Yes')"
-        ),
-        'Conflict_Any' => array(
+            'Type'        => "enum('No','Yes')",
+        ],
+        'Conflict_Any'                   => [
             'Description' => 'Conflict between any reviews',
-            'Type' => "enum('No','Yes')"
-        )
-    );
+            'Type'        => "enum('No','Yes')",
+        ],
+    ];
+    // phpcs:enable
```

Comment:
```suggestion
```

the disable was removed but the enable is still here

Reply:
Fixed.